frontend/javascripts/dashboard/dataset/dataset_settings_data_tab.tsx

```
@@ -554,21 +554,23 @@ function SimpleLayerForm({
                   rules={[
                     {
                       validator: (_rule, value) =>
-                        value == null || value === "" || (value > 0 && value < 2 ** bitDepth)
+                        value == null ||
+                        value === "" ||
+                        (value >= valueRange && value <= valueRange[1] && value != 0)
```

Biome (1.9.4)

[error] Use !== instead of !=. != is only allowed when comparing against `null`

!= is only allowed when comparing against null

Using != may be unsafe if you are relying on type coercion
Unsafe fix: Use !==


(lint/suspicious/noDoubleEquals)